JavaScript/JavaScriptSDK.Tests/CheckinTests/DataLossAnalyzer.tests.ts

```diff
@@ -89,4 +89,5 @@ class DataLossAnalyzerTests extends TestClass {
     }
 }
 
-new DataLossAnalyzerTests().registerTests(); 
\ No newline at end of file
+// Uncomment if you want to use DataLossanalyzer
+// new DataLossAnalyzerTests().registerTests(); 
```

Comment:
Yes, this requires a refactoring of DataLossAnalyzer (#208). I'll do that once we have this task done #289.